tests/Filtering/PossiblyUnusedClassesFilterTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TomasVotruba\ClassLeak\Tests\Filtering;
+
+use Iterator;
+use TomasVotruba\ClassLeak\Filtering\PossiblyUnusedClassesFilter;
+use TomasVotruba\ClassLeak\Tests\AbstractTestCase;
+use TomasVotruba\ClassLeak\ValueObject\FileWithClass;
+
+final class PossiblyUnusedClassesFilterTest extends AbstractTestCase
```

Comment:
I'd prefer to include this test as a single test fixture in the rule itselft, instead of adding a new unit test.
It would be easier to read for me later.

Reply:
I see. What I really want is the test to be simpler, as reading is quite complex now.

Reply:
On other hand, we should not encourage using wrong class casing. This should be spotted and trigger the class name fix instead.

Reply:
phpstan itself does already error about class-mismatches. I think its not the job of class-leak to report it as a problem

Reply:
Exactly, this tool builds on top of PHPStan best quality code. Let PHPStan handle these :+1: